app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -182,35 +182,35 @@ public void directionsApiCallBack(DirectionsResult result, List<Route> routeOpti
         setRoutesAdapter();
     }
 
-
     private void setRoutesAdapter() {
         // Android adapter for list view
         adapter = new RoutesAdapter(this, R.layout.list_routes, mViewModel.getRouteOptions());
         allRoutes.setAdapter(adapter);
         allRoutes.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
-                if(carButton.isSelected() || transitButton.isSelected() || walkButton.isSelected()){
+                if (carButton.isSelected() || transitButton.isSelected() || walkButton.isSelected()) {
                     Intent openPaths = new Intent(RoutesActivity.this,
                             PathsActivity.class);
                     DirectionsRoute directionsResult = mViewModel.getDirectionsResult().routes[i];
-                    openPaths.putExtra("directionsResult", directionsResult);
+                    openPaths.putExtra("directionsResult", directionsResult);                    openPaths.putExtra("shuttle", true);
```

Comment:
why is there a big space here?

Reply:
Oops probably by mistake it's supposed to be on the line under.. I'll fix it